analyzer/queries/queries.go

```diff
@@ -462,6 +462,27 @@ var (
 	    vote = excluded.vote,
       height = excluded.height;`
 
+	ValidatorBalancesUnprocessedEpochs = `
+    SELECT epochs.id, epochs.start_height
+    FROM chain.epochs as epochs
+    LEFT JOIN chain.validator_balance_history as history
+      ON epochs.id = history.epoch 
+    WHERE history.epoch IS NULL
+    ORDER BY epochs.id DESC
```

Comment:
I was thinking it'd be faster during the reindex to get useful info (since the api only cares about the last 24hrs), but now that we're constructing validator history chronologically this has been removed.